### PR TITLE
Darya Lapata | Portfolio 

### PR DESCRIPTION
**Pull request by DALA746**
https://daryalapata.netlify.app/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta property="og:title" content="Darya Lapata | Frontend Developer | Portfolio" />
+    <meta property="og:description" content="A portfolio and collection of project by Darya"/>
+    <meta property="og:image" content="https://daryalapata.netlify.app/img/profil.webp"/>
+    <meta property="og:url" content="https://daryalapata.netlify.app/"/>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <link rel="icon" href="img/star.png" type="image/png">
```
Comment (Rephili) [inline index.html]: Nice detail!

File: README.md [hunk 2]
```diff
@@ -1 +1,12 @@
-# portfolio
\ No newline at end of file
+# MY PORTFOLIO
+
+Thanks to that project I've learned a lot about CSS. First of all, everything is boxes and you can put more boxes in the boxes.
```
Comment (Rephili) [inline README.md]: Boxes and more boxes indeed :)

File: README.md [hunk 3]
```diff
@@ -1 +1,12 @@
-# portfolio
\ No newline at end of file
+# MY PORTFOLIO
+
+Thanks to that project I've learned a lot about CSS. First of all, everything is boxes and you can put more boxes in the boxes. 
+I've learned more about how to use Flexbox especially properties like flex-wrap and justify-content.
```
Comment (Rephili) [inline README.md]: Yeah, they are really useful! Haven't used CSS Grid at all in my projects, only Flexbox

File: index.html [hunk 4]
```diff
@@ -0,0 +1,356 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta property="og:title" content="Darya Lapata | Frontend Developer | Portfolio" />
```
Comment (Rephili) [inline index.html]: Great that you had the time to add og:tags

File: style.css [hunk 5]
```diff
@@ -0,0 +1,447 @@
...
+    width: 100%;
+    transition: .5s ease;
+    background-color:#bcbcbc;
+  }
+
+  .image-container:hover, .overlay {
+    opacity:0.7;
+  }
+
+  .text {
```
Comment (Rephili) [inline style.css]: Really like how this looks on your page :)

File: README.md [hunk 6]
```diff
@@ -1 +1,12 @@
...
\ No newline at end of file
+# MY PORTFOLIO
+
+Thanks to that project I've learned a lot about CSS. First of all, everything is boxes and you can put more boxes in the boxes. 
+I've learned more about how to use Flexbox especially properties like flex-wrap and justify-content. 
+
+I need to be better at planning my work before the start and think thought how my website should look from a CSS perspective. Maybe even skis on paper and have this planning visually. 
+At the beginning of this project, everything felt confusing to me, especially the CSS part. But at the end of the week, I am proud of what I've created.
+
+I've tested my wesite with Chrome Devtools Lighthouse and got accessibility of 81. Then I've fixed some issues with links by additing rel="noopener" to the a tag and converting all images to webp format ( https://convertio.co/)
```
Comment (Rephili) [inline README.md]: I tested it with https://validator.w3.org also (just to compare) and it looks good. It only complains a bit about how you use the h1s and that one section lacks heading (only warnings), so not really any problems! Good job!

**Review by Rephili — COMMENTED** [review]
Great project Darya! Your code is well structured with well named classes et.c., and you've also added comments in both the HTML and CCS file, that makes it even more readable. I like the fact that we both went for a nature/greenish theme. :)

File: README.md (hunk 2 above)
Comment (DALA746) [reply README.md]: Yes, but have to be careful not to overuse them :)

File: README.md (hunk 3 above)
Comment (DALA746) [reply README.md]: I like Flexbox more :)

**Comment by DALA746** [comment]
Thanks, Rebecca for your feedback! 💯  Yes, green is my favorite color :)

**Pull request closed without merging**